src/functions-templates/js/google-analytics/google-analytics.js

```diff
@@ -39,29 +39,24 @@ const proxyToGoogleAnalytics = function (event, done) {
   console.info('proxying params:', params)
   const qs = querystring.stringify(params)
 
-  const reqOptions = {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'image/gif',
-    },
-    url: GA_ENDPOINT,
-    body: qs,
+  try {
+    const response = await fetch(GA_ENDPOINT, {
+      method: 'POST',
+      headers: { 'Content-Type': 'image/gif' },
+      body: qs,
+    })
+    const { statusCode, statusMessage } = await response.json()
```

Comment:
Is `json` correct here? We weren't parsing the response before.
Also should this be `const { status, statusText } = response`?
Should we also check `response.ok`?

Reply:
Good point, I messed that up. Fixed!